libsolidity/analysis/StaticAnalyzer.cpp

```diff
@@ -163,7 +163,7 @@ bool StaticAnalyzer::visit(VariableDeclaration const& _variable)
 			m_errorReporter.warning(
 				3408_error,
 				_variable.location(),
-				"Variable covers a large part of storage and thus makes collisions likely. "
+				"Variable \"" + _variable.name() + "\" covers a large part of storage and thus makes collisions likely. "
```

Comment:
Don't we have an `utils` function that adds quotes around a string? Maybe @Marenz wrote it if I recall correctly

Reply:
@leonardoalt 
There is `solidity::util::escapeAndQuoteString`. I could add just `quote`, if we do not have it yet. Or we can use `'`, as I think clang does.

Reply:
I think `escapeAndQuoteString` was added because we didn't want `std::quote`, so I would use that.

Reply:
It also replaced all these "\"" + ... + "\"" cases then.